homework8/ErikaTatchyn/task1.js

```diff
@@ -0,0 +1,29 @@
+// Turn given object into iterable so it was possible to loop through it via for...of construction.
+// Given object:
+let range = {
```

Comment:
don't need `let` here. use `let` for that case, when you are going to rewrite that variable. Otherwise - `const`